src/rust/engine/fs/src/directory_tests.rs

```diff
@@ -33,6 +33,7 @@ fn assert_entry_is_err(tree: &DigestTrie, path: &str) {
 
 #[test]
 fn entry_simple() {
+    env_logger::init();
```

Comment:
I wouldn't expect this particular `entry_simple` test to trigger this error message. Could you explain a bit about why this is added here?

Thanks

Reply:
`env_logger::init` panics if called more than once but IMO it should be called at the start of unit testing. Is there an annotation that can be used to run this exactly once at the start of testing? ChatGPT says I could create an `initialize` function using an instance of `std::sync::Once` but I'd still need to call `initialize` at the start of every test that could output logging.

Reply:
Thanks for the background. I'm inclined to remove this before landing.

There's three factors in my mind:

- The test order is not deterministic: I think it runs in alphabetical order by default (so `entry_simple` isn't first), but also in parallel.
- As you say, `env_logger:init` can only be called once, so having any "surprise" calls buried in a test suite might lead to confusion in future.
- We're not asserting the contents of the log messages so having it here seems like it just helps with manual testing?

Thus, I think once we're happy with the result of manual testing, we should remove:

```suggestion
```

Reply:
Done